yodeploy/virtualenv.py

```diff
@@ -112,6 +122,35 @@ def create_ve(app_dir, pypi=None, req_file='requirements.txt'):
         os.chdir(cwd)
 
 
+def check_requirements(ve_dir, requirements):
+    """Given a ve root, and a list of requirements, ensure that they are met"""
+    script = os.path.join(ve_dir, 'bin', 'check_requirement')
+    if not os.path.exists(script):
+        with open(script, 'w') as f:
+            f.write(
+                '#!/usr/bin/env python\n'
```

Comment:
Yeah, it's a bit icky to leave this in the virtualenv. Esp when it is so minimalist.

I'm thinking maybe I'll pipe it into a python process, rather than write any file. Makes it harder to debug by hand, but I don't think that's a big deal....